### PR TITLE
Create issue template to cover common issues

### PR DESCRIPTION
**Pull request by callumacrae**
This is a bit more obvious than the "Please review the guidelines for contributing to this repository." banner - the user has to active delete the text to make an issue.

Also put in an actual template, which should help a bit for actual bug reports :)

@contra @phated: Thoughts?

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+This tracker is for bug reports only.
+
+
+Before opening an issue, please make sure you've checked the following:
+
+- For support requests, please use Stack Overflow (stackoverflow.com) or Gitter (see the README).
+
+- If the bug is in a plugin, open an issue on the plugin repository, not the gulp repository.
+
+- If you're getting a deprecated module warning, don't worry about it: we're aware of it and it's not an issue. To make it go away, update to Gulp 4.0.
```
Comment (callumacrae) [inline .github/ISSUE_TEMPLATE.md]: This should probably link the an issue with an explanation. Is there one for minimatch, or just for lodash?

**Comment by phated** [comment]
There's not a specific issue for minimatch. Feel free to create one or just link to a github search. I'll merge this now and it can be improved upon.

**Comment by phated** [comment]
Also, want to mention something about work being done on lots of dependencies, and gulp isn't dead or unmaintained?

**Comment by phated** [comment]
Nevermind, missed that point.

**Pull request merged**